carma/launch/environment.launch.py

```diff
@@ -399,6 +401,124 @@ def generate_launch_description():
             )
         ]
     )
+
+    # Cooperative Perception Stack
+    carma_cooperative_perception_container = ComposableNodeContainer(
+        package='carma_ros2_utils', # rclcpp_components
+        name='carma_cooperative_perception_container',
+        executable='carma_component_container_mt',
+        namespace= GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                package='carma_cooperative_perception',
+                plugin='carma_cooperative_perception::ExternalObjectListToDetectionListNode',
+                name='cp_external_object_list_to_detection_list_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': True}, 
+                    {'--log-level' : GetLogLevel('cp_external_object_list_to_detection_list_node', env_log_levels) },
+                    {'is_lifecycle_node': True} # Flag to allow lifecycle node loading in lifecycle wrapper
```

Comment:
These nodes are already lifecycle nodes, so why would they need to be wrapped in a lifecycle wrapper?

Reply:
removed all mentions of the extra tag. It works fine without it